lib/components/viewers/route-details.tsx

```diff
@@ -240,7 +243,7 @@ class RouteDetails extends Component<Props> {
                   onClick={() => this._stopLinkClicked(stop)}
                   onMouseOver={() => setHoveredStop(stop.id)}
                   routeColor={
-                    routeColor.includes('ffffff') ? '#333' : routeColor
+                    routeColor.includes('ffffff') ? grey[900] : routeColor
```

Comment:
Why 900 over the 800 used as the default route color in `connected-transit-vehicles`?